live-css-editor/src/input.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react'
+
+export default class Input extends Component {
+    render() {
+        return (
+            <form className='input-form'>
+                <label htmlFor="style">CSS Styles: </label>
+                <textarea cols='50' rows='20' onChange={e => this.props.changeStyle(e.target.value)}/>
+            </form>
```

Comment:
Related to the comment above, there is no way to add more HTML elements to test the styling. This could be very useful as the user can just copy and paste the HTML and test the styles.